blocks/components/editable/index.js

```diff
@@ -320,46 +326,62 @@ export default class Editable extends wp.element.Component {
 	}
 
 	render() {
-		const { tagName, style, value, focus, className, showAlignments = false, formattingControls } = this.props;
-		const classes = classnames( 'blocks-editable', className );
+		const {
+			tagName,
+			style,
+			value,
+			focus,
+			className,
```

Comment:
It's a little strange to me that when a parent component renders this one, the `className` is applied to a nested element, not the top-level element. Could we not just apply it to the `<div>` and update other styles to target the TinyMCE element?